File: fix_status.py
import re

with open('backend/services/rules_engine.js', 'r', encoding='utf-8') as f:
    engine = f.read()

engine = engine.replace('STATUS.PNOC', 'S.PNOC')
engine = engine.replace('STATUS.REVIEW', 'S.REVIEW')
engine = engine.replace('STATUS.PASS', 'S.PASS')
engine = engine.replace('STATUS.NA', 'S.NA')

with open('backend/services/rules_engine.js', 'w', encoding='utf-8') as f:
    f.write(engine)

File: fix_status.js
const fs = require('fs');
let code = fs.readFileSync('backend/services/rules_engine.js', 'utf8');

// Fix the return value of normalizeLegacyStatus
code = code.replace(/return 'estimated';/g, "return 'MANUAL REVIEW';");

// Fix the hardcoded status strings
code = code.replace(/status: 'estimated'/g, "status: 'MANUAL REVIEW'");

fs.writeFileSync('backend/services/rules_engine.js', code);
console.log("STATUS FIXED");
